src/app/game/page.tsx

```diff
@@ -7,7 +7,7 @@ import Viewport from "@/components/viewport"
 import { generateRandomPiece } from "@/utils/pieces"
 import { useState } from "react"
 
-const Game = () => {
+function Game() {
```

Comment:
please describe why this is an improvement

Reply:
Its much prettier 😁